java-api-examples/src/websocketsrv/AsrWebsocketServer.java

```diff
@@ -0,0 +1,254 @@
+/*
+ * // Copyright 2022-2023 by zhaoming
+ */
+// java websocketServer
+// usage: AsrWebsocketServer soPath modelCfgPath
+package websocketsrv;
+
+import com.k2fsa.sherpa.onnx.OnlineRecognizer;
+import com.k2fsa.sherpa.onnx.OnlineStream;
+import java.io.*;
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.net.UnknownHostException;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.nio.FloatBuffer;
+import java.util.*;
+import java.util.Collections;
+import java.util.concurrent.*;
+import java.util.concurrent.LinkedBlockingQueue;
+import org.java_websocket.WebSocket;
+import org.java_websocket.drafts.Draft;
+import org.java_websocket.drafts.Draft_6455;
+import org.java_websocket.handshake.ClientHandshake;
+import org.java_websocket.server.WebSocketServer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * AsrWebSocketServer has three threads pools, one pool for network io, one pool for asr stream and
+ * one pool for asr decoder.
+ */
+public class AsrWebsocketServer extends WebSocketServer {
+  private static final Logger logger = LoggerFactory.getLogger(AsrWebsocketServer.class);
+  //  Queue between io network io thread pool and stream thread pool, use websocket as the key
+  static LinkedBlockingQueue<WebSocket> streamQueue = new LinkedBlockingQueue<WebSocket>();
```

Comment:
Is there a reason to use a static member here?

Reply:
static maybe more efficiency and one srv instance assumption. I can remove it if needed